packages/block-editor/src/components/link-control/style.scss

```diff
@@ -97,37 +99,9 @@ $preview-image-height: 140px;
 	order: 20;
 }
 
-.block-editor-link-control__search-results-wrapper {
-	position: relative;
-	margin-top: -$grid-unit-20 + 1px;
-
-	&::before,
-	&::after {
-		content: "";
-		position: absolute;
-		left: -1px;
-		right: $grid-unit-20; // avoid overlaying scrollbars
-		display: block;
-		pointer-events: none;
-		z-index: 100;
-	}
-
-	&::before {
-		height: $grid-unit-20 * 0.5;
-		top: 0;
-		bottom: auto;
-	}
-
-	&::after {
-		height: $grid-unit-20;
-		bottom: 0;
-		top: auto;
-	}
-}
-
 .block-editor-link-control__search-results {
-	margin: 0;
-	padding: $grid-unit-20 * 0.5 $grid-unit-20 $grid-unit-20 * 0.5;
+	margin-top: -$grid-unit-20;
```

Comment:
I think the usage of a negative margin here necessitates a comment to explain _why_ we're doing it. @richtabor can you recall why you added this?

Reply:
I added a comment. It's needed because the link control field needs margin all around it to render properly without search results.